### ISSUE TITLE
Text Record Presentation Header Includes Space For Row Number Column

### ISSUE THREAD

**Issue by jetzerb** [dbeaver/dbeaver#16521]
#### System information: 
- DBeaver version 22.0.4.202205011839

#### Describe the problem you're observing:
"Record" view text presentation header incorrectly includes space for row number.

#### Steps to reproduce, if exist:
Window >> Preferences >> Editors >> Data Editor >> Appearance >> Plain text >> Check the "Show line numbers" box.

Run this script and switch to Text mode (`` CTRL+` ``):
```sql
select * from (values (1,'one','I'),(2,'two','II')) t("number",written,roman);
```
Results:
```
| # | number | written | roman |
+---+--------+---------+-------+
| 1 |      1 | one     | I     |
| 2 |      2 | two     | II    |
```
Now switch to "Record" mode (`Tab`) and see that the header/data delineation line still includes space for the row number (the leading `+---`):
```
| Name    | Value |
+---+---------+-------+
| number  | 1     |
| written | one   |
| roman   | I     |
```

**Comment by LonwoLonwo** [comment]
Thanks for the bug report @jetzerb

**Comment by Destrolaric** [comment]
@jetzerb, Hi. Is this still reproducible in the recent version of DBeaver?

**Comment by jetzerb** [comment]
@Destrolaric : No, it's no longer an issue.  Not sure when it was fixed, but I'm currently running 22.2.5 and the text displays correctly.
